packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -31,6 +31,7 @@ import {
 // See /packages/api/src/routes/index.ts for reasoning and instructions to add new routes
 
 export type BlockId = RootHex | Slot | "head" | "genesis" | "finalized";
+export const mimeTypeSSZ = "application/octet-stream";
```

Comment:
This already exists for state downloads. Do you think we should move this to a CONST and pull both from the same place?

https://github.com/ChainSafe/lodestar/blob/97dfcfc45c098e4a1363bb0213695916eeb94487/packages/api/src/beacon/routes/debug.ts#L26